installation/language/en-GB/en-GB.ini

```
@@ -110,6 +110,7 @@ INSTL_COMPLETE_ADMIN_BTN="Complete & Open Admin"
 INSTL_COMPLETE_FINAL="Installation is Complete"
 INSTL_COMPLETE_FINAL_DESC="Your Joomla installation is now complete and ready to use."
 INSTL_COMPLETE_ADD_PRECONFIG="Customise Installation"
+INSTL_REMOVE_INST_FOLDER="Are you absolutely sure you want to delete the installation folder? The folder cannot be restored afterwards!"
```

let's be consistent with other similar strings please

```suggestion
INSTL_REMOVE_INST_FOLDER="Are you sure you want to delete? Confirming will permanently delete the installation folder"
```